### DISCUSSION TITLE
Hiring new peoples

### DISCUSSION THREAD

**Discussion by RiteshBalayan** [RiteshBalayan/instaorbit#8] [General]
I think we find new people to join the team. We have big work to do and idea about Space, Physics and potential AI, Is good enough to attract high calibre people.

**Comment by hrithikthakur** [comment]
> I think we find new people to join the team. We have big work to do and idea about Space, Physics and potential AI, Is good enough to attract high calibre people.

ok. right now or later?

**Reply by RiteshBalayan** [reply]
Now!
